src/Microsoft.AspNetCore.ApplicationInsights.HostingStartup/JavaScriptSnippetTagHelperComponent.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.ApplicationInsights.AspNetCore;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.ApplicationInsights.HostingStartup
+{
+    /// <summary>
+    /// The <see cref="TagHelperComponent"/> that injects the <see cref="JavaScriptSnippet"/> at the end of the head tag.
+    /// </summary>
+    public class JavaScriptSnippetTagHelperComponent : TagHelperComponent
+    {
+        private readonly string _javaScriptSnippet;
+
+        /// <summary>
+        /// Initializes the <see cref="JavaScriptSnippetTagHelperComponent"/>.
+        /// </summary>
+        /// <param name="javaScriptSnippet">The <see cref="JavaScriptSnippet"/> to inject in the head tag.</param>
+        public JavaScriptSnippetTagHelperComponent(JavaScriptSnippet javaScriptSnippet)
+        {
+            _javaScriptSnippet = javaScriptSnippet.FullScript;
```

Comment:
Will `FullScript` have `{0}` replaced with the instrumentation key? Or it will happen someplace else?

Reply:
https://github.com/Microsoft/ApplicationInsights-aspnetcore/blob/12cb0cd96c83115eeb16d7b2711c5ddb73c9e69c/src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs#L54 it's replaced.